src/net/sourceforge/kolmafia/textui/parsetree/CompositeValue.java

```diff
@@ -45,7 +45,7 @@
 import org.json.JSONException;
 import org.json.JSONObject;
 
-public class CompositeValue
+public abstract class CompositeValue
 	extends Value
```

Comment:
It seems strange to have an abstract class without any abstract methods (even if it's technically legal). Are there any methods that subclasses always implement (and always need specialized logic for)?

Just skimming for methods that have useless stub implementations, clear / keys / aref / aset / remove / dumpValue?